testdata/config/scylla.yaml

```diff
@@ -8,5 +8,4 @@ client_encryption_options:
   keyfile: /etc/scylla/db.key
   truststore: /etc/scylla/ca.crt
   require_client_auth: true
-# when using 5.4.x we have to specify force_schema_commit_log option
-force_schema_commit_log: true
+maintenance_socket: workdir
```

Comment:
The addition of `maintenance_socket` here is not related to the commit description. Why is it here?

Reply:
You are right it should be in the second commit